Makefile

```diff
@@ -94,15 +97,15 @@ run:  ## Runs the site like it runs in production
 
 .PHONY: dev
 dev: prereqs ## Runs the site as a developer; including live reload support and installation of git hooks
-	NODE_OPTIONS="$(TS_NODE_ARGS) $(NODE_ARGS)" ./node_modules/.bin/supervisor -w app.ts,lib,etc/config,static/tsconfig.json -e 'js|ts|node|properties|yaml' -n exit --exec $(NODE) -- ./app.ts $(EXTRA_ARGS)
```

Comment:
TSX automatically watches any file in the import graph, so we shouldn't have to list out everything anymore, only the "extra" files.

Reply:
"any file in the import graph" -> including configuration files?

Reply:
oh!!! I see above the `etc/config` :)